### PR TITLE
release 0.10.4

### PR DESCRIPTION
**Pull request by dkent600**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+## 0.10.5
+  - Features Added
```
Comment (orenyodfat) [inline CHANGELOG.md:2]: 0.10.5?

Comment (dkent600) [reply CHANGELOG.md:2]: that is one of the things the outstanding changelog PR fixes

**Comment by orenyodfat** [comment]
the daos search feature does not work good.
e.g try to look for dxdao at https://alchemy-dev-ptvuxnws4rd8foboex.herokuapp.com/ @tibetsprague ?

**Comment by dkent600** [comment]
@orenyodfat I can find dxDAO no problem.  Can you please be more specific?

@tibetsprague  The problem I see is with searching for Genesis Alpha.  Once you search for it it never goes away.  And if you search twice it always shows up twice.  Always.  Three times... four....

**Comment by dkent600** [comment]
@orenyodfat

The process to follow is:

1) answer here whether each found problem must be fixed in order to release.
2) create tickets for everything you find, regardless.

**Pull request merged**